src/worker.h

```diff
@@ -34,6 +37,7 @@ class WorkerServiceImpl final : public WorkerService::Service {
   Status ExecuteTask(ServerContext* context, const ExecuteTaskRequest* request, ExecuteTaskReply* reply) override;
   Status ImportFunction(ServerContext* context, const ImportFunctionRequest* request, ImportFunctionReply* reply) override;
   Status Die(ServerContext* context, const DieRequest* request, DieReply* reply) override;
+  Status ImportReusableVariable(ServerContext* context, const ImportReusableVariableRequest* request, AckReply* reply) override;
```

Comment:
I intended it to parallel `ImportFunction` since in both cases things are flowing from the driver to the workers through the scheduler. We aren't "importing" anything in the sense of the Python keyword "import", but we are importing something in the sense of bringing it in from the outside, which is how I interpreted the "import" in "ImportFunction".

I don't mind renaming it, but I like the idea of it being the same as ImportFunction.